src/client/app/about/about.component.scss

```diff
@@ -0,0 +1,13 @@
+:host {
```

Comment:
Lets keep it as a CSS file; by using `scss` we implicitly force developers to use Sass.